CMakeLists.txt

```diff
@@ -207,3 +209,8 @@ add_subdirectory(src/opentimelineio)
 if(OTIO_PYTHON_INSTALL)
     add_subdirectory(src/py-opentimelineio)
 endif()
+
+if(OTIO_CXX_EXAMPLES AND OTIO_PYTHON_INSTALL)
+    add_subdirectory(examples)
+endif()
```

Comment:
Should this write an error if `OTIO_CXX_EXAMPLES` is on but `OTIO_PYTHON_INSTALL` is off?  Or a message of some kind?

Reply:
Actually now that the C++/Python code is in separate examples, maybe this should be changed to just:
```
if(OTIO_CXX_EXAMPLES)
    add_subdirectory(examples)
endif()
```
And then the examples/CMakeLists.txt could optionally add the C++/Python code when Python is enabled:
```
if(OTIO_PYTHON_INSTALL)
    list(APPEND examples python_adapters_child_process)
    list(APPEND examples python_adapters_embed)
endif()
```
That way the pure C++ examples can be built without Python.

Reply:
Ah yeah -- lets do that.  That sounds simpler and clearer.  Thanks!